front/components/assistant/conversation/lib.ts

```diff
@@ -71,6 +71,7 @@ export function createPlaceholderUserMessage({
       profilePictureUrl: image,
       timezone: Intl.DateTimeFormat().resolvedOptions().timeZone ?? "UTC",
       username,
+      origin: "web",
```

Comment:
Couldn't we infer this in the private endpoint?

Reply:
I'd rather be explicit + as we build more product suites this will likely become in the future "conversations" "triggers" etc... so might have well have it in the signature in as many place as possible.

Reply:
Not fully convinced by the footprint of this but won't block.

Reply:
Any reason though why it's not in the URL? So it can also be used to debug where the load is coming from on those endpoints?

Reply:
That would be weird to put it in the POST URL no?